src/js/utils/DOMUtils.js

```diff
@@ -1,6 +1,26 @@
 const HEIGHT_ATTRIBUTES = ['paddingTop', 'paddingBottom', 'borderTopWidth', 'borderBottomWidth'];
 const WIDTH_ATTRIBUTES = ['paddingLeft', 'paddingRight', 'borderLeftWidth', 'borderRightWidth'];
 
+let matchesFn = function () {
```

Comment:
Another minor #nit but lets wrap this function in parenthesis so its easier to identify as a self executing function.